element_lab/lab.py

```diff
@@ -79,59 +103,13 @@ class ProtocolType(dj.Lookup):
 class Protocol(dj.Lookup):
     definition = """
     # protocol approved by some institutions like IACUC, IRB
-    protocol                : varchar(16)
+    protocol                : varchar(36)
```

Comment:
it's okay to use a shorter name for the protocol primary key with a full name in a secondary attribute.